extensions/picocli/deployment/src/main/java/io/quarkus/picocli/deployment/PicocliDeploymentConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.picocli.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "picocli")
+class PicocliDeploymentConfiguration {
+
+    /**
+     * Whether or not a native image processing build steps should be enabled.
+     * This should be set to false if `picocli-codegen` annotation processor is used.
+     */
+    @ConfigItem(name = "native-image.processing.enable", defaultValue = "true")
+    boolean nativeImageProcessingEnabled;
```

Comment:
As mentioned in my comment above, I am very reluctant about this...

Reply:
As per my previous comment, the existence of this property is important to me. Please revert this change. 🙏

Reply:
I think it's the wrong thing to do, but I won't fight it :). Feel free to add it again

Reply:
Thank you!

Reply:
No need to thank me 😊. We are just trying to figure out what's best for users 😎